src/MagnumPlugins/WavAudioImporter/Test/WavImporterTest.cpp

```diff
@@ -278,6 +312,10 @@ void WavImporterTest::mono32f() {
 
     CORRADE_COMPARE(importer.format(), Buffer::Format::MonoFloat);
     CORRADE_COMPARE(importer.frequency(), 48000);
+
+    CORRADE_COMPARE_AS(importer.data().slice(0, 16),
+        Containers::Array<char>::from(0, 0, 0, 0, 108, 57, -103, 59, 3, 63, 42, 60, -33, -81, -120, 60).slice(0, 16),
+        TestSuite::Compare::Container);
```

Comment:
@mosra http://www.ietf.org/rfc/ien/ien137.txt

It used to be a lot more prevalent, but I work on machines that use different endianness for floats and ints. I don't know why.